src/Uno.UI/UI/Xaml/Controls/Border/BorderLayerRenderer.Android.cs

```diff
@@ -160,10 +168,14 @@ internal void Clear()
 						// Related Issue: https://github.com/unoplatform/uno/issues/6893
 						using (var strokePaint = new Paint(borderBrush.GetStrokePaint(drawArea)))
 						{
-							//Create the path for the outer and inner rectangles that will become our border shape
-							var borderPath = cornerRadius.GetOutlinePath(drawArea.ToRectF());
-							borderPath.AddRoundRect(adjustedArea.ToRectF(), cornerRadius.GetInnerRadii(borderThickness), Path.Direction.Cw);
+							//Create the path for the outer and inner rectangles that will become our border shape							
+							using var borderPath = new Path();
 
+							borderPath.AddRoundRect(drawArea, _outerRadiiStore, Path.Direction.Cw);
+							borderPath.AddRoundRect(adjustedArea, _innerRadiiStore, Path.Direction.Cw);
```

Comment:
Oh interesting, I would have thought that the two paths needed to be "intersecting", like one should be clockwise and the other should be counter-clockwise. I remember that the use of a gradient brush was causing problems with the EvenOdd Fill rule: https://github.com/unoplatform/uno/blob/7b86f1dc1734d9cd4bb53fab8b1be8195303e3c3/src/Uno.UI/UI/Xaml/Controls/Border/BorderLayerRenderer.iOSmacOS.cs#L402-L403

Reply:
Seems it is not problematic in case of Android for some reason :-D . 
![image](https://user-images.githubusercontent.com/1075116/201695404-098491c9-51ac-4816-8b6f-6dd9e9a56f8e.png)

Reply:
Cool! Good to know.